app/src/services/cpk.ts

```diff
@@ -325,6 +340,52 @@ class CPKService {
       throw err
     }
   }
+
+  redeemPositions = async ({
+    isConditionResolved,
+    oracle,
+    questionId,
+    numOutcomes,
+    marketMaker,
+    conditionalTokens,
+  }: CPKRedeemParams): Promise<TransactionReceipt> => {
+    try {
+      const transactions = []
+      if (!isConditionResolved) {
+        transactions.push({
+          operation: CPK.CALL,
+          to: oracle.address,
+          value: 0,
```

Comment:
I'm starting to wonder if it wouldn't make sense to have `operation: CPK.CALL` and `value: 0` as default values. @cag how do you feel about that?